client/src/actions/index.js

```diff
@@ -189,7 +189,7 @@ export const putData = (
         payload: response.data,
       });
       if (directTo.length > 3) {
```

Comment:
why are you checking if length of `directTo` is > 3?

Reply:
directTo is the next location after an action has been performed, so am checking if the directTo value length is more than 3. if so it directs user to the next page while if it is not it does not direct user to another page